lib/configure_logging_for_browser.dart

```diff
@@ -48,43 +48,45 @@ class ConfigureLoggingForBrowser {
   }
 
   static void listenJsErrors(LoggingService loggingService,
-      {bool preventDefault: true, html.Window window, Protector infiniteLoopProtector}) {
+    {bool preventDefault: true, html.Window window, Protector infiniteLoopProtector, bool ignoreJsErrors}) {
```

Comment:
I think, inverted option would be better. For example, handleGlobalJsErrors.

Besides, it should be the default value for the argument to prevent "Boolean expression must not be null" assertion error  in the following condition.

Reply:
setUpAll function has default value and set it into listenJsErrors. That's right?

Reply:
`setUpAll` is a particular consumer of the method and uses the correct arguments, but it isn't the only one. In other words, `listenJsErrors` is a public method and it fails, if you aren't pass the argument.